controller/src/main/java/io/pravega/controller/task/Stream/TaskStepsRetryHelper.java

```diff
@@ -20,8 +20,8 @@
 class TaskStepsRetryHelper {
     private static final long RETRY_INITIAL_DELAY = 100;
     private static final int RETRY_MULTIPLIER = 2;
-    private static final int RETRY_MAX_ATTEMPTS = 100;
-    private static final long RETRY_MAX_DELAY = Duration.ofSeconds(5).toMillis();
+    private static final int RETRY_MAX_ATTEMPTS = 10;
```

Comment:
What is the rationale behind reducing max attempts for retrying task steps. This will cause stream creation at bootstrap to fail with higher probability if segment stores do not come up within few seconds of controller start.